http/src/main/java/com/hazelcast/jet/contrib/http/impl/HttpListenerSinkContext.java

```diff
@@ -137,14 +118,56 @@ private void putToBuffer(T item) {
         boolean hasConnectedClients();
     }
 
+    interface MessageBuffer {
```

Comment:
Why don't we mark this and all other interfaces and classes in this class `private`?

Reply:
I've seen your approve and just merged the PR withtout addressing this. good idea, I'll do it in a separate PR.